src/devhub/entity/addon/kanban/post_ticket.jsx

```diff
@@ -181,65 +139,92 @@ const KanbanPostTicket = ({ metadata }) => {
       </div>
     ) : null;
 
+  const showDescription = post_type === "Comment";
+  const showFunding = features.approved_sponsorship_value;
+  const showSponsor = features.sponsorship_supervisor;
+  const showTags = Array.isArray(tags) && features.tags;
+
+  if (isTableView) {
```

Comment:
We can also rmove this part now

Reply:
Actually Ori asked to keep the code for now, because we will have build on this for different addon.

Reply:
I have removed it as well